src/latex/conway-ledger.tex

```diff
@@ -34,8 +34,8 @@
 
 \appendix
 
-\input{Ledger/EssentialAgda}
-\input{Ledger/PDF/ConwayBootstrapEnact}
-\input{Ledger/PDF/ConwayBootstrap}
+\input{Conway/Ledger/EssentialAgda}
+\input{Conway/Ledger/PDF/ConwayBootstrapEnact}
+\input{Conway/Ledger/PDF/ConwayBootstrap}
 
```

Comment:
Moving the tex files generated for Conway to a separate directory allows for simpler build rules.  Otherwise, we have to be more careful about duplication in the Makefile and redefinition of build rules.